repo/README.md

```diff
@@ -57,7 +57,19 @@ Below are examples, your milage may vary.
 
 ### Integration into Eclipse
 
-TBD - should be similar to IntelliJ above, using Eclipse external tools, using the `${resource_loc}` variable for the absolute path to the currently selected file or folder
+Setup external tools under `Run > Exernal Tools > Exernal Tool Configurations...`.
+Add a Programm which points to the `repo` shell script for each desired REPO operation. 
+I would recommend to disable `Build before launch` in the Build tab.
+
+Examples:  
+* put
+  * Location: `/Applications/Dev/tools/repo-1.1/repo`
+  * Working Directory: `${container_loc}`
+  * Arguments: `put -f ${selected_resource_name}`
+* get
+  * Location: `/Applications/Dev/tools/repo-1.1/repo`
+  * Working Directory: `${container_loc}`
+  * Arguments: `get -f ${selected_resource_name}`
```

Comment:
Will this work for folders (i.e. selecting folders in the tree and running the external tool to push an entire subtree)? Because in Intellij it didn't really work, so I had to use the absolute path to the file (`${resource_loc}` in Eclipse AFAICS), which works fine with the repo tool, the working directory doesn't matter.